lib/polishgeeks/dev-tools/command/final_blank_line.rb

```diff
@@ -0,0 +1,105 @@
+module PolishGeeks
+  module DevTools
+    module Command
+      # Validator used to check if all files have final blank line
+      class FinalBlankLine < Base
+        self.type = :validator
+
+        attr_reader :counter
+
+        # Default paths which we want to exclude from analyse
+        DEFAULT_PATHS_TO_EXCLUDE = %w(
+          coverage tmp log vendor public
+          app/assets/images
+          app/assets/fonts
+        )
+
+        # Executes this command and set output and counter variables
+        def execute
+          @output = []
+          @counter = 0
+
+          files_to_analyze.each do |file|
+            @counter += 1
+            @output << sanitize(file) if File.size(file) > 0 && IO.readlines(file).last[-1] != "\n"
```

Comment:
could you extract this: File.size(file) > 0 && IO.readlines(file).last[-1] != "\n" into a separate method?